desktop/src/main/java/bisq/desktop/main/dao/economy/dashboard/BsqDashboardView.java

```diff
@@ -122,32 +128,37 @@ private BsqDashboardView(DaoFacade daoFacade,
 
     @Override
     public void initialize() {
-
```

Comment:
please only change stuff that actually changes something. Makes it more cumbersome to review...